pipeline/clean/opuscleaner/configs/default.filters.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
Thought: It's a shame this is JSON as we can't add comments about the configuration and why certain defaults were chosen.

Reply:
I created this filter template using the OpusCleaner UI tool. It should be compatible with the [filters](https://github.com/hplt-project/OpusCleaner/tree/main/opuscleaner/filters). I don't know if they have a schema somewhere to validate it. This would be a good follow-up task to validate the schema before running things.